Common/src/main/java/com/blamejared/crafttweaker/natives/entity/ExpandEntityType.java

```diff
@@ -27,6 +29,7 @@
 @ZenRegister
 @Document("vanilla/api/entity/EntityType")
 @NativeTypeRegistration(value = EntityType.class, zenCodeName = "crafttweaker.api.entity.EntityType")
+@TaggableElement(value = "minecraft:entity_type", managerFactoryClass = EntityTypeTagFactory.class)
```

Comment:
As discussed on Discord, it might be better to have two annotations here: one for `@TaggableElement` and one for `@TagManagerFactoryFor`